aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -32,99 +34,96 @@ static void parallel_apply(Tensor& result, const Tensor& self, F f) {
 
   static tbb::affinity_partitioner ap;
 
-  auto arr_out = result.data<scalar_t>();
-  auto arr_in = self.data<scalar_t>();
-  int64_t size = self.numel();
-  if (size < internal::TBB_GRAIN_SIZE) {
-    unary_kernel(arr_out, arr_in, size, f);
-  } else {
-    tbb::parallel_for(
-        tbb::blocked_range<int64_t>(0, size, internal::TBB_GRAIN_SIZE),
-        [&](const tbb::blocked_range<int64_t>& r) {
-          auto size = r.end() - r.begin();
-          unary_kernel(arr_out + r.begin(), arr_in + r.begin(), size, f);
-        },
-        ap);
-  }
+  // if (result.is_contiguous() && self.is_contiguous()) {
+  //   auto arr_out = result.data<scalar_t>();
+  //   auto arr_in = self.data<scalar_t>();
+  //   int64_t size = self.numel();
+  //   if (size < internal::TBB_GRAIN_SIZE) {
+  //     unary_kernel(arr_out, arr_in, size, f);
+  //   } else {
+  //     tbb::parallel_for(
+  //         tbb::blocked_range<int64_t>(0, size, internal::TBB_GRAIN_SIZE),
+  //         [&](const tbb::blocked_range<int64_t>& r) {
+  //           auto size = r.end() - r.begin();
+  //           unary_kernel(arr_out + r.begin(), arr_in + r.begin(), size, f);
+  //         },
+  //         ap);
+  //   }
+  // } else {
```

Comment:
Dead code?

Reply:
This is for testing only. This commit provides a correct baseline for all perf optimizations.